README.md

```diff
@@ -1,3 +1,39 @@
 # Ada Developers Academy Jump Start
 
-This repository contains the materials presented during the Ada Developers Academy jump start program. This program is intended for accepted Ada Developers Academy students to give them a jump start on learning materials and concepts that will be utilized throughout our year-long program.
+This repository contains the materials presented during the [Ada Developers Academy](adadevelopersacademy.org) jump start program. This program is intended for accepted Ada Developers Academy students to give them a jump start on learning materials and concepts that will be utilized throughout our year-long program.
+
+## Learning Goals
+_Problem Solving_  
+Breaking down complex problems is a fundamental skill as a software engineer. While we will be doing this for years to come, we want to give you an opportunity to explore problem solving in programatic ways an exploring topics like:
+- input
+- output
+- algorithms
+- variable substitution
+
+_Learning Style_  
+Whether you have gone through a traditional post-secondary program recently or not, coming to the Ada classroom can be an adjustment for many folks. It is good to get a gauge on your learning style as soon as possible to be able to jump right in on day one and absorb the material in a way that works well for you.
+
+_Workflow_  
+Learning to become a programmer is based a lot in using your computer in the most efficient way possible so you can focus on the difficult problems rather than navigating your computer.
+
+_Exposure to Programming_  
+Our year-long program will go from basics to advanced topics throughout the seven month curriculum. To give everyone an similar jump-off point, we want to ensure that all students have been exposed to programming and programming concepts including:
+- variables
+- data structures
+- substitution
+- patterns
+
+## Table of Contents
+| Week    | Day                | Topic
+|---------|--------------------|-----------------------------------------
+| Week 1 | [Day 1](#week-1)  | Problem Solving
+| Week 1 | [Day 1](#week-1)  | Workflow Navigation
+| Week 1 | [Day 2](#week-1)  | Google!
+| Week 1 | [Day 2](#week-1)  | Terminal
+| Week 2 | [Day 1](#week-1)  | Big Picture
+| Week 2 | [Day 1](#week-1)  | Environment Setup
+| Week 2 | [Day 1](#week-1)  | Hello World
+| Week 2 | [Day 1](#week-1)  | Variables, Data Types, User Input/Output
+| Week 2 | [Day 2](#week-1)  | Programming Expressions
+| Week 3 | [Day 1](#week-1)  | Arrays, Hashes
+| Week 3 | [Day 2](#week-1)  | Iterators
```

Comment:
Hmm... I don't know about the time suggestions (Week/Day), what if we just say "Lesson 1", "Lesson 2", etc...?